kubernetes/charts/weblogic-operator/templates/_operator-dep.tpl

```diff
@@ -19,9 +19,16 @@ spec:
   replicas: 1
   template:
     metadata:
-     labels:
+      {{- with .annotations }}
+      annotations:
```

Comment:
should annotations: be before line 22?

Reply:
No, I'm only going to include the line "annotations:" if there is at least one annotation given by the customer. This is different from "labels:" where the chart already has some labels it will add by default.